slides/bitcoin_applications.tex

```diff
@@ -0,0 +1,491 @@
+				\centering
+				\includegraphics[width=1.5cm]{../assets/images/diploma.png}\\
+				University issues\\
+				diploma
+				\end{column}
+			\begin{column}{0.05\textwidth}
+				\includegraphics[width=0.75cm]{../assets/images/big_arrow.png}
+			\end{column}
+			\begin{column}{0.3\textwidth}
+				\centering
+				\includegraphics[width=1.5cm]{../assets/images/fingerprint.png}\\
+				Computes hash value of diploma
+			\end{column}
+			\begin{column}{0.05\textwidth}
+				\includegraphics[width=0.75cm]{../assets/images/big_arrow.png}
+			\end{column}
+			\begin{column}{0.3\textwidth}
+				\centering
+				\includegraphics[width=1.5cm]{../assets/images/write_onchain.png}\\
+				Saves hash value\\ on blockchain
+			\end{column}
+		\end{columns}
+	\end{minipage}\vfill
+	\vspace{0.5cm}
+	\begin{minipage}{\linewidth}
+	\textbf{\large{Verification:}}
+	\begin{columns}
+		\begin{column}{0.3\textwidth}
+			\centering
+			\includegraphics[width=1.5cm]{../assets/images/receive_diploma.png}\\
+			Potential employer receives diploma
+		\end{column}
+		\begin{column}{0.05\textwidth}
+			\includegraphics[width=0.75cm]{../assets/images/big_arrow.png}
+		\end{column}
+		\begin{column}{0.3\textwidth}
+			\centering
+			\includegraphics[width=1.5cm]{../assets/images/fingerprint.png}\\
+			Computes hash value of diploma
+		\end{column}
+		\begin{column}{0.05\textwidth}
+			\includegraphics[width=0.75cm]{../assets/images/big_arrow.png}
+		\end{column}
+		\begin{column}{0.3\textwidth}
+			\centering
+			\includegraphics[width=1.5cm]{../assets/images/verify_onchain.png}\\
+			Compares with on-chain hash value
+		\end{column}
+	\end{columns}
+\end{minipage}
+\end{frame}
+
+%%%
+
+\begin{frame}{Proof of Existence}
+	\begin{columns}
+		\begin{column}{0.4\textwidth}
+			\begin{figure}
+				\centering
+				\includegraphics[width=4cm, frame]{../assets/images/Diploma_unibas.PNG}
+				\scriptsize{\href{https://cif.unibas.ch/en/events-projects/certificates/}{cif.unibas.ch/en/}}
+			\end{figure}
+		\end{column}
+		\begin{column}{0.6\textwidth}
+			\begin{itemize}
+				\item<1 -> Certificate for Blockchain-related courses since 2018 
+				\item<2 -> Roll-out for entire University of Basel coming soon
+			\end{itemize}
+		\end{column}
+	\end{columns}
+\end{frame}
+
+%%%
+
+\begin{frame}{Commit-Reveal}
+	\begin{itemize}
+		\item<1 -> Long known cryptographic primitive
+		\item<2 -> First, commitment to a choice, without revealing it.
+		\item<3 -> In second phase, choice is revealed.
+		\item<4 -> Important: Commitment is secret and binding.\vspace{0.5cm}
+		\item<5 -> Example:
+		\begin{itemize}
+			\item<5 -> Alice and Bob want to play rock-paper-scissors
+			\item<6 -> Playing at the exact same time is difficult on blockchains
+			\item<7 -> Solution: To commit to their choices, they hash them (salted) and save the hash value on-chain
+			\item<8 -> Revealing their choices and the salts they can then determine the winner in a second step
+		\end{itemize}
+	\end{itemize}
+\end{frame}
```

Comment:
Do students already understand what a "salt" is? Maybe this example should be more in-depth.

Reply:
I thought that this could be explained in this example, as it is shown in the JSON passport and in the merkle tree.